Description/ru-RU.md

```diff
@@ -0,0 +1,79 @@
+# Русский (Россия)
```

Comment:
You can remove the country if there are no regional variants of Russian.

Reply:
Well, makes sense
```suggestion
# Русский
```